src/components/molecules/FormFields/FilesInput/index.tsx

```diff
@@ -26,6 +26,8 @@ export default function FilesInput(props: InputProps): ReactElement {
         setIsLoading(true)
 
         if (field.name === 'serviceSelfDescription') {
+          props?.setStatus && props.setStatus('loading')
```

Comment:
question(blocking): Why don't we write it as `props?.setStatus('loading')` and remove the first check?